src/stories/20_Features/Animations.html

```diff
@@ -212,6 +222,111 @@ <h1>whoosh-in-right</h1>
         </div>
       </amp-story-grid-layer>
     </amp-story-page>
+    
+    <!-- ## Zoom-In -->
+    <!-- You can also use animations on images to create visual effects. -->
+    <!-- 
+    <div class="ampstart-card info">
+      **Tip:** Create a Ken Burns effect by combining these! Learn how in the [visual effects section.](/stories/visual_effects/ken_burns/)
+    </div>
+    -->
+    <amp-story-page id="zoom-in">
+      <amp-story-grid-layer template="vertical">
+        <h1>zoom-in</h1>
+        <div class="spacer"></div>
+        <amp-img animate-in="zoom-in"
+                 animate-in-duration="4s"
+                 layout="responsive"
+                 src="https://picsum.photos/1024/768?image=1026"
+                 width="720"
+                 height="320">
+        </amp-img>
+      </amp-story-grid-layer>
+    </amp-story-page>
+
+    <!-- ## Zoom-Out -->
+    <amp-story-page id="zoom-out">
+      <amp-story-grid-layer template="vertical">
+        <h1>zoom-out</h1>
+        <div class="spacer"></div>
+        <amp-img animate-in="zoom-out"
+                 animate-in-duration="4s"
+                 layout="responsive"
+                 src="https://picsum.photos/1024/768?image=1026"
+                 width="720"
+                 height="320">
+        </amp-img>
+      </amp-story-grid-layer>
+    </amp-story-page>
+
+    <!-- ## Pan-Left -->
+    <!--
+    <div class="ampstart-card info">
+      **Tip:** When using the `fill` template of the `amp-story-grid-layer` only the first element will be displayed. Wrap elements inside a `<div>` to display everything inside it.
+    </div>
+     -->
+    <amp-story-page id="pan-left">
+      <amp-story-grid-layer template="fill">
+        <div>
+          <h1>pan-left</h1>
```

Comment:
Stupid question: would it be more idiomatic to move the title into a different layer? e.g.:

```
<amp-story-grid-layer template="vertical">
   <h1>zoom-out</h1>
</amp-story-grid-layer>
```

Reply:
We could do it, but with a couple of caveats:

1. Ordering-wise in the markup, the layer where the title is located would have to be after the layer with the image, since the last "fill" template layer in an amp-story-page will be the element located at the top of the stacking context. 
2.  We would still need to wrap the amp-img in a div, even if it's the only element in the layer - since the "fill" template will change the dimensions of its first child, we need a div to "catch" those style changes. Only then will the image have its dimensions unchanged and the animations will work as intended.

I'll commit the changes to the PR so you can see how the code would look like.

Reply:
Ah - thanks for explaining!

In this sample I'd prefer a simpler markup using a vertical layout:

```
    <amp-story-page id="pan-left">
      <amp-story-grid-layer template="vertical">
        <h1>pan-left</h1>
        <amp-img animate-in="pan-left"
                 animate-in-duration="4s"
                 layout="fixed"
                 src="https://picsum.photos/720/320?image=1026"
                 width="720"
                 height="320">
        </amp-img>
      </amp-story-grid-layer>
    </amp-story-page>
```

The animations (in particular for pan-left and pan-right) are more visible as well (they move only a few pixels with the fill layout). The reason why I'm so picky about this is that this a sample demonstrating the animations, additional markup such as the div is distracting in this case. Wdyt?

Reply:
No problem!

Regarding the change to a vertical layout, if we take into account
[this new change on the runtime side](https://github.com/ampproject/amphtml/pull/14334), which will scale up the size of the image if it's smaller than the screen size, the image will sit on top of the title. Additionally, there will be some padding on the sides added by the vertical layout. (See screenshot). We could use a z-index to make the title be on top of the image, but not sure what you think about the padding. Let me know what you think.

![image](https://user-images.githubusercontent.com/5449100/38369561-02e16858-38b6-11e8-9680-8eb3335d7c5a.png)

![image](https://user-images.githubusercontent.com/5449100/38369617-2cc030e6-38b6-11e8-9602-4ad5b09b2e6e.png)